### PR TITLE
Collection of small changes resulting from final removal of `plasmapy_sphinx` from `plasmapy`

### PR DESCRIPTION
**Pull request by rocco8773**
1. fix typos
2. copy over CSS changes

### CODE REVIEW

**Review by namurphy — APPROVED** [review]

**Comment by rocco8773** [comment]
**TL;DR** We can NOT have the version flyout in the sidebar for our theme at the moment.  It will have to live on the page like any theme that is not `sphinx_rtd_theme`.

@namurphy I think I have all the last changes needed from the migration from `plasmapy`.  However, I've discovered an "issue" that is too big of a rabbit hole to try to "fix" at this time.  This rabbit hole is associated with the [version flyout menu](https://docs.readthedocs.io/en/stable/flyout-menu.html).

As you're probably aware of, the flyout menu is generated in the sidebar when using `sphinx_rtd_theme` but gets placed on the bottom-right corner of the page when not using `sphinx_rtd_theme`.  `sphinx_rtd_theme` does this using some JS magic and I found hack to make it work for our theme, but this hack does not work if we have [RTD's (Beta) Addons](https://github.com/readthedocs/addons) enabled, which we do for `plasmapy`. 

I tried implementing Addon's tag [`readthedocs-embed-flyout`](https://docs.readthedocs.io/en/stable/flyout-menu.html#defining-where-the-flyout-menu-is-injected) to inject the flyout, but this is poorly documented and does not appear to do anything during my prototyping.

Question: Do we want to live with the flyout location and continue with implementing `plasmapy_theme` into `plasmapy`?  I think we should at least deploy `plasmapy_sphinx` as is.

**Comment by namurphy** [comment]
> @namurphy I think I have all the last changes needed from the migration from `plasmapy`. However, I've discovered an "issue" that is too big of a rabbit hole to try to "fix" at this time. This rabbit hole is associated with the [version flyout menu](https://docs.readthedocs.io/en/stable/flyout-menu.html).

Another Sphinx rabbit hole with a dragon in it... 🐰 🕳️ 🐉 

> Question: Do we want to live with the flyout location and continue with implementing `plasmapy_theme` into `plasmapy`? I think we should at least deploy `plasmapy_sphinx` as is.

This is preserving the status quo, I think?  I agree we should deploy `plasmapy_sphinx` as is, and save the fixes to this problem for later.

Also: thank you so for doing this!

**Comment by namurphy** [comment]
Also I'm planning on doing a release tomorrow or Wednesday (the deadline for including the newest version in the Python in Heliophysics Community summer school).  Would it be okay to merge the full removal of `plasmapy_sphinx` after I do that release?  I'm thinking it'd be safer to delay it in case this change causes any more Sphinx rabbit dragon holes... (or maybe Sphinx rabbit hole dragons?)

The official release of `plasmapy_sphinx` does not need to wait, though.

**Comment by rocco8773** [comment]
Yes, go ahead and release `plasmapy` without the removal of `plasmapy_sphinx`.

**Comment by rocco8773** [comment]
> > Question: Do we want to live with the flyout location and continue with implementing `plasmapy_theme` into `plasmapy`? I think we should at least deploy `plasmapy_sphinx` as is.
> 
> This is preserving the status quo, I think? I agree we should deploy `plasmapy_sphinx` as is, and save the fixes to this problem for later.

Yes and no.  Yes, this is how the flyout has behaved for non-`sphinx_rtd_theme`'s since the beginning of time. No, we have been using `sphinx_rtd_theme` in `plasmapy`, so our flyout will change locations when we implement `plasmapy_theme`.
